payjoin-cli/src/app.rs

```diff
@@ -616,6 +653,88 @@ impl App {
     }
 }
 
+#[cfg(feature = "v2")]
+struct SendStore {
+    req_ctx: Option<payjoin::send::RequestContext>,
+    file: std::fs::File,
+}
+
+#[cfg(feature = "v2")]
+impl SendStore {
+    fn new() -> Result<Self> {
```

Comment:
maybe we should export a trait to cover the impls we have in both `SendStore` and `ReceiveStore`.. could make it easier for developers

Reply:
Perphaps `payjoin-io` could become an public API in the future even including default `ureq` implementations. This would be a higher level abstraction than `payjoin`